config/toml.go

```diff
@@ -535,6 +535,27 @@ max_open_connections = {{ .Instrumentation.MaxOpenConnections }}
 
 # Instrumentation namespace
 namespace = "{{ .Instrumentation.Namespace }}"
+
+#######################################################
+###              Remote Event Collection            ###
+#######################################################
+[event_collector]
+
+# The URL of the influxdb instance to use for remote event 
+# collection. If empty, remote event collection is disabled.
+url = "{{ .EventCollector.URL }}"
+
+# The influxdb token to use for remote event collection.
+token = "{{ .EventCollector.Token }}"
+
+# The influxdb bucket to use for remote event collection.
+bucket = "{{ .EventCollector.Bucket }}"
+
+# The influxdb org to use for event remote collection.
+org = "{{ .EventCollector.Org }}"
+
+# The size of the batches that are sent to the database.
+batch_size = {{ .EventCollector.BatchSize }}
```

Comment:
Can this work locally as well?

Reply:
yeah, its really easy and fun to spin up your own instance and play around. I've been using this repo to spin up local influx db instances

https://github.com/guessi/docker-compose-grafana-influxdb